bool low_voltage
bool overloaded
int8 power      # PWM duty cycle -100 ... 100
float32 encoder # degree
float32 RPM   	# degree per second
float32 ODO	    # odometry
float32 speed   # mm per second
